explore-cc.php

```diff
@@ -0,0 +1,121 @@
+<?php
+  echo "
+      <script type=\"text/javascript\">
+        function toggleCCExplore() {
+          var element = document.getElementById('cc-explore'); element.classList.toggle('is-active');
+        }
+      </script>
+  ";
+?>
```

Comment:
```suggestion
<script type="text/javascript">
function toggleCCExplore() {
  var element = document.getElementById('cc-explore'); element.classList.toggle('is-active');
}
</script>
```

Reply:
Correct me if I'm wrong, but I believe the above suggestion should work without further modifying the `heder.php`.

Reply:
Yes. I just tested it.